cmd/vic-machine/configure/configure.go

```diff
@@ -217,6 +238,39 @@ func updateSessionEnv(sess *executor.SessionConfig, envName, envValue string) {
 	sess.Cmd.Env = newEnvs
 }
 
+func (c *Configure) processCertificates(conf *config.VirtualContainerHostConfigSpec) error {
```

Comment:
In theory, you should not need value from *config.VirtualContainerHostConfigSpec to process any input parameters